app/src/test/java/mollie/tictactoe/testdoubles/ContextDummy.java

```diff
@@ -0,0 +1,532 @@
+package mollie.tictactoe.testdoubles;
+
+import android.content.BroadcastReceiver;
+import android.content.ComponentName;
+import android.content.ContentResolver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.IntentSender;
+import android.content.ServiceConnection;
+import android.content.SharedPreferences;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.content.res.AssetManager;
+import android.content.res.Configuration;
+import android.content.res.Resources;
+import android.database.DatabaseErrorHandler;
+import android.database.sqlite.SQLiteDatabase;
+import android.graphics.Bitmap;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.os.UserHandle;
+import android.support.annotation.Nullable;
+import android.view.Display;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class ContextDummy extends Context {
```

Comment:
@felipesere I did start using Mockito, but have since abandoned it.  I agree that it's hard to see if any methods are different, but I was hoping that by naming it `Dummy` would relay that this class does nothing and shouldn't be returning anything useful?  I'm happy to change it back to Mockito if you think that's not enough?